aptos-move/e2e-testsuite/src/tests/loader.rs

```diff
@@ -47,4 +47,19 @@ proptest! {
     ) {
         run_and_assert_universe(universe, additional_txns);
     }
+
+    #[test]
+    fn smaller_world_and_test_deps_charging(
```

Comment:
We might also want to test module publishing. It's not urgent though so maybe just leave a TODO?